src/System.Private.ServiceModel/tools/scripts/WCFCustomCleanup.cmd

```diff
@@ -0,0 +1,36 @@
+@if "%_echo%" neq "on" echo off
```

Comment:
I originally designed to be only called from build.cmd. Now I think Dev can use it to launch startwcfservice.cmd. 

In the clean up, it's usually the best effort - output error as warning, then continue.

What are other conditions you observed other than assume CertificateGenerator is around?